zanata-war/src/main/java/org/zanata/ZanataInit.java

```diff
@@ -257,7 +257,7 @@ private void checkLuceneLocks(File indexDir)
             // we don't trust Lucene's NativeFSLockFactory for NFS locks
             String docURL = "http://docs.jboss.org/hibernate/search/4.4/reference/en-US/html/search-configuration.html#search-configuration-directory-lockfactories";
             log.warn("The Hibernate Search index dir '{}' might be NFS. " +
-                    "NativeFSLockFactory might not be safe: {}" +
+                    "NativeFSLockFactory might not be safe: See {}",
                     indexDir, docURL);
```

Comment:
@carlosmunoz the important change was replacing '+' with ','.  It was printing the arguments out of order, and it was really confusing until I got to the bottom of it.